bundle/store.go

```diff
@@ -608,6 +682,47 @@ func hasRootsOverlap(ctx context.Context, store storage.Store, txn storage.Trans
 	return nil
 }
 
+func applyPatches(ctx context.Context, store storage.Store, txn storage.Transaction, patches []PatchConfig) error {
+	for _, pat := range patches {
+
+		// construct patch path
+		path, ok := patch.ParsePatchPathEscaped("/" + strings.Trim(pat.Path, "/"))
+		if !ok {
+			return fmt.Errorf("error parsing patch path")
+		}
+
+		var op storage.PatchOp
+		switch pat.Op {
+		case "add":
+			op = storage.AddOp
+
+			_, err := store.Read(ctx, txn, path[:len(path)-1])
+			if err != nil {
+				if !storage.IsNotFound(err) {
+					return err
+				}
+
+				if err := storage.MakeDir(ctx, store, txn, path[:len(path)-1]); err != nil {
+					return err
+				}
+			}
```

Comment:
Instead of changing the behaviour of `add` (from how it is defined in JSON Patch) WDYT about supporting this kind of operation w/ another name (`upsert`) like we do in decision log masks: https://www.openpolicyagent.org/docs/latest/management-decision-logs/#masking-sensitive-data

In either case, let's make sure we describe the behaviour in the docs as it's not standard JSON Patch.

Reply:
Changed `add` -> `upsert` and added a section in the docs about this.